### DISCUSSION TITLE
Test dependencies in org.eclipse.tm4e.core?

### DISCUSSION THREAD

**Discussion by jensli** [eclipse/tm4e#526] [Q&A] [answered]
Hello, text mates!

org.eclipse.tm4e.core contains both implementation and tests.

It declares no dependencies on JUnit in its MANIFEST.MF.

Because of this it doesn't compile for me in the IDE. The test classes reports "The import org.junit cannot be resolved".

How is this supposed to work? How do I get the IDE to build the test classes in org.eclipse.tm4e.core?

**Comment by sebthom** [comment, marked as answer]
The .project file references the Junit5 classpath container:
https://github.com/eclipse/tm4e/blob/ac34c5f440f95e1046ecb69b1c3c5c8514cb4a86/org.eclipse.tm4e.core/.classpath#L14

I see it like this in Eclipse:
![image](https://user-images.githubusercontent.com/426959/236278736-3a9b60c5-8d1e-47a2-898a-b6dcd2ce6fa7.png)

I have the following plugins installed in eclipse:
![image](https://user-images.githubusercontent.com/426959/236278244-65c9b874-984d-47c6-aa3f-125224e00a3a.png)

**Reply by jensli** [reply]
Yes it does! Somehow I had managed to delete that classpath entry. I am very confused. Thanks!
